workers/vita/Dockerfile

```diff
@@ -17,56 +17,77 @@ RUN sed 's/^deb \(.*\)/deb-src \1/' /etc/apt/sources.list \
 ENV VITASDK=/opt/vitasdk
 ENV PATH=$PATH:$VITASDK/bin
 
-ARG URL=https://github.com/vitasdk/autobuilds/releases/download/master-linux-v567/vitasdk-x86_64-linux-gnu-2017-10-29_09-48-44.tar.bz2
+#Install VitaSDK base
```

Comment:
Comments should explain “why”, they should never describe “what”. This is true for all computer programming all the time. All of the comments that have been added are describing “what”, which is useless, the code already says what it is doing.